dist/profile/templates/kubernetes/resources/jenkinsio/configmap.yaml.erb

```diff
@@ -56,36 +56,34 @@ data:
       rewrite ^/census/(.*) https://census.jenkins.io/$1 permanent;
       rewrite ^/jenkins-ci.org.key$ https://pkg.jenkins.io/redhat/jenkins.io.key permanent;
   
+      # TODO: Migrate to jenkins.io redirects
       # permalinks
       # - this one is referenced from 1.395.1 "sign post" release
-      rewrite ^/why$            https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=53608972 permanent;
+      rewrite ^/why$            https://www.jenkins.io/ permanent;
       # baked in the help file to create account on Oracle for JDK downloads
       rewrite ^/oracleAccountSignup$    http://www.oracle.com/webapps/redirect/signon?nexturl=http://jenkins-ci.org/ permanent;
-      # to the donation page
-      rewrite ^/donate$        https://wiki.jenkins-ci.org/display/JENKINS/Donation permanent;
       # CLA links used in the CLA forms
-      rewrite ^/license$        https://wiki.jenkins-ci.org/display/JENKINS/Governance+Document#GovernanceDocument-cla permanent;
-      rewrite ^/licenses$        https://wiki.jenkins-ci.org/display/JENKINS/Governance+Document#GovernanceDocument-cla permanent;
+      rewrite ^/license$        https://www.jenkins.io/project/governance/#cla permanent;
+      rewrite ^/licenses$        https://www.jenkins.io/project/governance/#cla permanent;
       # used to advertise the project meeting
-      rewrite ^/meetings/$        https://wiki.jenkins-ci.org/display/JENKINS/Governance+Meeting+Agenda permanent;
+      rewrite ^/meetings/$        https://www.jenkins.io/project/governance-meeting/ permanent;
       # used from friends of Jenkins plugin to link to the thank you page
-      rewrite ^/friend$        https://wiki.jenkins-ci.org/display/JENKINS/Donation permanent;
+      rewrite ^/friend$        https://www.jenkins.io/donate/#friend-of-jenkins permanent;
       # used by Gradle JPI plugin to include fragment
       rewrite ^/gradle-jpi-plugin/latest$    https://raw.github.com/jenkinsci/gradle-jpi-plugin/master/install permanent;
```

Comment:
This link to Gradle JPI plugin is dead. CC @sghill for advice

Reply:
Thanks Oleg.

This install file was removed in https://github.com/jenkinsci/gradle-jpi-plugin/commit/914f1934a1cd7eeadf91195017c673f6b686ebc4 in 2015.

I think it'd be clearest if the url returned a HTTP 410 GONE.

#### Context
The install script was meant to be applied to a build script like this:
```groovy
apply from:"https://raw.github.com/jenkinsci/gradle-jpi-plugin/master/install"
```
If we return a 410, Gradle fails with a clear error message:
```
* What went wrong:
A problem occurred evaluating root project 'my-plugin'.
> Could not get resource '<url>'.
   > Could not GET '<url>'. Received status code 410 from server: Gone
```

I considered redirecting to the README, but it would fail in a less clear way:
```
* What went wrong:
Could not compile script 'https://raw.github.com/jenkinsci/gradle-jpi-plugin/master/README.md'.
> startup failed:
  script 'https://raw.github.com/jenkinsci/gradle-jpi-plugin/master/README.md': 1: expecting '!', found ' ' @ line 1, column 2.
     # Gradle JPI plugin
      ^
  
  1 error
```

Reply:
There is no support for "gone" on jenkins.io at the moment unfortunately. Would be great to add it indeed